server/src/tenant/query-builder/interfaces/record.interface.ts

```diff
@@ -1,8 +1,5 @@
 export interface Record {
-  id?: string;
```

Comment:
cc @magrinj let me know if you are ok with that!

Reply:
@Weiko I think we can keep them, as we're still putting them manually, so we're sure there are here at least for all records